api/views.py

```diff
@@ -44,517 +53,157 @@ def api_root(request):
-        source_id                     string                  guid of associated OSF object (e.g. node_id for an OSF project)
-        url                           string                  url of associated OSF object (e.g. project url)
-        metadata                      object                  additional information about the item
-        date_created                  iso8601 timestamp       date/time when the item was created
-        date_submitted                iso8601 timestamp       date/time when the item was submitted
-        date_accepted                 iso8601 timestamp       date/time when the item was accepted
-        location                      string                  location of the event item
-        start_time                    iso8601 timestamp       date/time when the event item begins
-        end_time                      iso8601 timestamp       date/time when the event item ends
-        category                      string                  item category (e.g. 'talk', 'poster')
-
-    ## Actions
-
-    ### Creating New Items
-
-            Method:        POST
-            URL:           /api/collections/<collection_id>/items OR /api/meetings/<meeting_id>/items
-            Query Params:  <none>
-            Body (JSON):   {
-                             "data": {
-                               "type": "items",                  # required
-                               "attributes": {
-                                 "title":       {title},         # required
-                                 "description": {description},   # optional
-                                 "type":        {type},          # required
-                                 "status":      {status},        # required
-                                 "source_id":   {source_id},     # optional
-                                 "url":         {url},           # optional
-                                 "metadata":    {metadata},      # optional
-                                 "location":    {location},      # optional
-                                 "start_time":  {start_time},    # optional
-                                 "end_time":    {end_time},      # optional
-                                 "category":    {category}       # required
-                               }
-                             }
-                           }
-            Success:       201 CREATED + item representation
-
-    Note: Items added by the collection creator will automatically have the status "approved". If "approve_all" is true
-    in collection.settings, items added by other users will automatically have the status "approved", otherwise
-    they will be "pending".
-
-    #This Request/Response
+    # `ItemViewSet`
     """
+
+    queryset = Item.objects.all()
     serializer_class = ItemSerializer
-    permission_classes = (drf_permissions.IsAuthenticatedOrReadOnly, )
-    pagination_class = LargeResultsSetPagination
```

Comment:
Ditto for this, this is why we don't have to make 10 separate requests to populate the schedule